hometask_day_2/task5.py

```diff
@@ -0,0 +1,20 @@
+my_string = "hello 1 one two three 15 world"
+
+def find_three_words_in_a_row(string):
+    str_to_search = string.split()
+    counter = 0
+    for word in str_to_search:
+
+        if counter == 3:
+            return "There are 3 words in a row"
+
+        try:
+            num = int(word)
```

Comment:
in `try` section just left `int(word)` and remove any other code. No need again specify `count = 0` and unused `n` variable